src/Showpad/Client.php

```diff
@@ -163,13 +163,32 @@ public function assetsTagsAddById($id, $tag)
 
         // Create request
         $data = $this->auth->request(
-            'GET',
+            'LINK',
             $resource,
             $parameters
         );
 
         return $data;
     }
+    
+    /**
+     * Find tag id by tag name
+     *
+     * POST /assets/{id}/tags.json
+     *
+     * @param string $tag The tag name
+     *
+     * @return tag id
+     */
+    public function getTagByName($tag)
```

Comment:
As far as I know, we have the `tagsList` method, which speaks to exactly the same endpoint, which might be an indication that we just need an extra parameter there? If you only want the first result back, you can decide for yourself in your application, instead of doing the `$data['response']['items']['0']['id'];` in this package.